crates/matrix-sdk-store-encryption/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "matrix-sdk-store-encryption"
 version = "0.1.0"
 edition = "2021"
+authors = ["Damir Jelić <poljar@termina.org.uk>"]
```

Comment:
I don't think we should add authors fields. It's not required for publishing and git history (and possibly copyright headers) is much better at tracking authorship.

Reply:
```suggestion
```